packages/backend/src/open_payments/payment/incoming/routes.test.ts

```diff
@@ -122,7 +120,7 @@ describe('Incoming Payment Routes', (): void => {
         },
         {
           incomingPaymentId: uuid(),
-          accountId: account.id
+          accountId: paymentPointer.id
```

Comment:
Is this because the OP spec still refers to accountId?

Reply:
Correct. Ditto for https://github.com/interledger/rafiki/pull/434#discussion_r952683467